src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const arrayOfWomen = (!withChildren)
```

Comment:
You can simplify the ternary operator by removing the parentheses around `!withChildren`. Use `const arrayOfWomen = !withChildren ? ... : ...` instead.